ln_init: deterministic channel IDs failure

Error looks like this:

```
LNInit   - Waiting for 78 channel opens in mempool...
LNInit   - Confirmed 78 channel opens in block 300
LNInit   - Checking deterministic channel IDs in block...
LNInit   - Assertion failed
Traceback (most recent call last):
  File "/shared/archive.pyz/test_framework/test_framework.py", line 131, in main
    self.run_test()
  File "/shared/archive.pyz/ln_init.py", line 270, in run_test
    assert block_txs[ch["id"]["index"]] == ch["txid"], (
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Actual txid:ac3ee9267ff1bc1ba47fc48cf95e53c31ddfcc82163f718f5ea985b3b6dcf6e6
{'capacity': 1000000, 'id': {'block': 300, 'index': 3}, 'push_amt': 500000, 'source_policy': {'cltv_expiry_delta': 40, 'fee_base_msat': 0, 'fee_proportional_millionths': 2000, 'htlc_maximum_msat': 990000000, 'htlc_minimum_msat': 1000}, 'target': 'tank-0019-ln', 'target_policy': {'cltv_expiry_delta': 40, 'fee_base_msat': 0, 'fee_proportional_millionths': 750, 'htlc_maximum_msat': 990000000, 'htlc_minimum_msat': 1000}, 'source': 'tank-0018-ln', 'txid': '17c88fc3914742ef36cc0769eba9e935e76d44d527935e6629b35b21ff075d9e'}
```

Background: when we init a LN in warnet, we create each channel open TX with a decreasing fee rate, so we can predict where that TX will be in the block and therefore can predict the short channel ID which is just a series of integers `block height || tx index || output index` and we use this to make the warnet channel graph match the input JSON.

There is some kind of issue with the block in this example. One of the TXs got ordered incorrectly.

index in block | txid | fee | size (vbytes) | weight | actual rate | expected rate
-|-|-|-|-|-|-
1 | cb57c0 | 0.00823914 | 164 | 658 | 5023.8 | 5001
2 | 85bd25 | 0.00823091 | 165 | 658 | 4988.4 | 4996
3 | ac3ee9 | 0.00817324 | 164 | 656 | 4983.6 | 4961
4 | 17c88f | 0.00822267 | 165 | 658 | 4983.4 | 4991

The issue is that `17c88f` was **supposed** to be index 3 and `ac3ee9` was **supposed** to be ... index 9! (not even close)

```
LNInit   - Sending channel open from tank-0018-ln -> tank-0019-ln with fee_rate=4991
LNInit   - Channel open tank-0018-ln -> tank-0019-ln
  outpoint=17c88fc3914742ef36cc0769eba9e935e76d44d527935e6629b35b21ff075d9e:0
  expected channel id: {'block': 300, 'index': 3}
...
LNInit   - Sending channel open from tank-0008-ln -> tank-0044-ln with fee_rate=4961
LNInit   - Channel open tank-0008-ln -> tank-0044-ln
  outpoint=ac3ee9267ff1bc1ba47fc48cf95e53c31ddfcc82163f718f5ea985b3b6dcf6e6:0
  expected channel id: {'block': 300, 'index': 9}
```


@cndolo as a short term temporary work around you can just disable the check that is failing in the scenario:

https://github.com/bitcoin-dev-project/warnet/blob/7cb394e2000fc498541ce2bd72cfed11b4df72e5/resources/scenarios/ln_init.py#L263-L273

This may mean that the channel policies applied to the graph in warnet are not a perfectly accurate match to the mainnet sample imported in the JSON

--- a/resources/scenarios/ln_init.py
+++ b/resources/scenarios/ln_init.py
@@ -311,7 +311,13 @@ class LNInit(Commander):
 
         def update_policy(self, ln, txid_hex, policy, capacity):
             self.log.info(f"Sending update from {ln.name} for channel with outpoint: {txid_hex}:0")
-            res = ln.update(txid_hex, policy, capacity)
+            res = None
+            while res is None:
+                try:
+                    res = ln.update(txid_hex, policy, capacity)
+                    break
+                except Exception:
+                    sleep(1)
             assert len(res["failed_updates"]) == 0, (
                 f" Failed updates: {res['failed_updates']}\n txid: {txid_hex}\n policy:{policy}"
             )

--- a/resources/scenarios/ln_init.py
+++ b/resources/scenarios/ln_init.py
@@ -240,11 +240,11 @@ class LNInit(Commander):
 
             channels = sorted(ch_by_block[target_block], key=lambda ch: ch["id"]["index"])
             index = 0
-            fee_rate = 5006  # s/vB, decreases by 5 per tx for up to 1000 txs per block
+            fee_rate = 5006  # s/vB, decreases by 20 per tx for up to 250 txs per block
             ch_threads = []
             for ch in channels:
                 index += 1  # noqa
-                fee_rate -= 5
+                fee_rate -= 20
                 assert index == ch["id"]["index"], "Channel ID indexes are not consecutive"
                 assert fee_rate >= 1, "Too many TXs in block, out of fee range"
                 t = threading.Thread(target=open_channel, args=(self, ch, fee_rate))

--- a/src/warnet/graph.py
+++ b/src/warnet/graph.py
@@ -356,7 +356,7 @@ def _import_network(graph_file_path, output_path):
         }
         tanks[source]["lnd"]["channels"].append(channel)
         index += 1
-        if index > 1000:
+        if index > 250:
             index = 1
             block += 1
         count += 1

